### DISCUSSION TITLE
Is there a way to include a local script using <Script>?

### DISCUSSION THREAD

**Discussion by Vacilando** [gatsbyjs/gatsby#37126] [Help]
I am including a script from unpkg.com using <Script> and that works fine.

But I have a local copy of the same script in a plugin under node_modules.

How can I include (what path) the local script?

Tried `<Script src='./node_modules/plugin/dist/xyz.pkgd.min.js' />` but that did not work.

**Comment by LekoArts** [comment]
Have you tried this?

```jsx
import YourScript from "plugin/dist/xyz.pkgd.min.js"

// Rest of component...
<Script src={YourScript} />
```

So use webpack to import the script and then use that. Because the node_modules path isn't available in the browser.

**Comment by Vacilando** [comment]
Thanks for the suggestion @LekoArts!

What works:

`<Script src='https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js' />`

Tried:

```
import maLayout from 'masonry-layout/dist/masonry.pkgd.min.js'

...

export const myComponent = ({ data }) => {
const isBrowser = typeof window !== 'undefined' 
...
  return (

    {isBrowser && <Script src={maLayout} />}

  )
...
```

but in develop getting:

```
 ERROR #11616 

There was an error while trying to create the client-only shell for displaying SSR errors:
ReferenceError: window is not defined

File: node_modules/masonry-layout/dist/masonry.pkgd.min.js:9:240
```

Any idea what am I doing wrong?

**Reply by LekoArts** [reply]
It's trying to access `window` during SSR which won't work. Maybe try this then:

```js
import maLayout from '!!raw-loader!masonry-layout/dist/masonry.pkgd.min.js'
```

**Comment by Vacilando** [comment]
Thanks, tried that, there is no error in develop or in browser console, but the Masonry functionality does not work then.

It's not too urgent to get this to work, would just be nice to use the local script (installed using npm install masonry-layout) instead of always fetching it from outside.
